### DISCUSSION TITLE
can't get PIN since 3.9.0 over HOOBS

### DISCUSSION THREAD

**Discussion by Sebyrem** [colinbendell/homebridge-blink-for-home#106] [Q&A]
Hello

I'm an hoobs user and before last update 3.9.0 my blink cameras were working well.

Since updating I can't reach my cameras anymore, trying to re install it all I can't get PIN ?

What do I have to do to get it all works as before ?

Seb

**Comment by bwilberg** [comment]
Hi Seb,
I had the same problem with iobroker and the ham module. I installed homebridge-blink-for-home version 3.7.6 and it is working again.
Best regards,
Bernd
